### DISCUSSION TITLE
WPF中的DataTable 怎么冻结首列或者首行。

### DISCUSSION THREAD

**Discussion by ccq1** [BYJRK/DotNet-Discussions#145] [WPF 开发]
就像Excel 文档一样，可以冻结首列。
![image](https://github.com/BYJRK/DotNet-Discussions/assets/78813459/f531e5bf-47a8-4d36-a775-592b94c8a18a)

**Comment by CKernal** [comment]
如果是DataGrid可以设置FrozenColumnCount试试

**Reply by ccq1** [reply]
好的谢谢解答，我回去试一试
